usaspending_api/references/management/commands/loadcfda.py

```diff
@@ -85,10 +105,12 @@ def load_cfda(fullpath):
                 if row['Archived Date']:
                     cfda_program.archived_date = datetime.strptime(row['Archived Date'], '%b, %d %Y')
 
-                cfda_program.save()
+                # TODO: add way to check/print out any cfda codes that got updated (not just created)
+                if created:
+                    logger.info('Created a new cfda code for {} ({})'.format(row['Program Number'],
+                                                                             row['Program Title']))
 
-                # self.logger.log(20, "loaded %s %s ", cfda_program.program_number, cfda_program)
+                cfda_program.save()
 
     except IOError:
-        logger = logging.getLogger('console')
-        logger.log("Could not open file to load from")
+        logger.info("Could not open file to load from")
```

Comment:
Suggestion:
`logger.info("Could not open file to load from {}".format(cfda_abs_path))`